sys-libs/hwloc/hwloc-1.11.8.recipe

```diff
@@ -0,0 +1,98 @@
+SUMMARY="The Hardware Locality library"
+DESCRIPTION="The Hardware Locality (hwloc) software project aims at easing the \
+process of discovering hardware resources in parallel architectures. It offers \
+command-line tools and a C API for consulting these resources, their locality, \
+attributes, and interconnection. hwloc primarily aims at helping \
+high-performance computing (HPC) applications, but is also applicable to any \
+project seeking to exploit code and/or data locality on modern computing \
+platforms."
+HOMEPAGE="http://www.open-mpi.org/projects/hwloc/"
+COPYRIGHT="1998-2011 Free Software Foundation, Inc."
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="http://www.open-mpi.org/software/hwloc/v1.11/downloads/hwloc-${portVersion}.tar.bz2"
+CHECKSUM_SHA256="ecef10265084d9123dc0b03d82ff5ee06fbf5ba755a912e7511ae590c9ee594d"
+PATCHES="hwloc-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	hwloc$secondaryArchSuffix = $portVersion
+	lib:libhwloc$secondaryArchSuffix = $portVersion
+	cmd:lstopo = $portVersion
+	cmd:lstopo_no_graphics = $portVersion
+	cmd:hwloc_annotate = $portVersion
+	cmd:hwloc_assembler = $portVersion
+	cmd:hwloc_assembler_remote = $portVersion
+	cmd:hwloc_bind = $portVersion
+	cmd:hwloc_calc = $portVersion
+	cmd:hwloc_compress_dir = $portVersion
+	cmd:hwloc_diff = $portVersion
+	cmd:hwloc_distances = $portVersion
+	cmd:hwloc_distrib = $portVersion
+	cmd:hwloc_dump_hwdata = $portVersion
+	cmd:hwloc_gather_topology = $portVersion
+	cmd:hwloc_info = $portVersion
+	cmd:hwloc_ls = $portVersion
+	cmd:hwloc_patch = $portVersion
+	cmd:hwloc_ps = $portVersion
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	pciutils$secondaryArchSuffix
+	lib:libltdl$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	hwloc${secondaryArchSuffix}_devel = $portVersion
+	devel:libhwloc$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	hwloc${secondaryArchSuffix} == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	libxml2_devel
+	libtool
+	"
+
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:grep
+	cmd:ld$secondaryArchSuffix
+	cmd:pkg_config
+	cmd:make
+	cmd:sed
+	cmd:autom4te
+	cmd:aclocal_1.15
+	cmd:autoconf
+	cmd:automake
+       "
+
```

Comment:
Add `defineDebugInfoPackage` macro for `lib:libhwloc`.

See updated [Guidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines).